ciao_contrib/_tools/fileio.py

```diff
@@ -953,36 +953,52 @@ def expand_evtfiles_stack(instack, pattern="*evt*"):
 # Do we need to use MJD; can we not just use TSTART?
 #
 def sort_mjd(filenames):
-    "Return filenames, sorted by MJD_OBS, as a list"
+    """Return filenames, sorted by MJD_OBS or MJD-OBS, as a list
+
+    The keyword used to be MJD_OBS but was changed to MJD-OBS
+    circa CIAO 4.12.
+    """
 
     if len(filenames) < 2:
         return list(filenames)
 
     vals = []
-    dmkeypar = runtool.make_tool('dmkeypar')
+
+    # This used to use dmkeypar to get the keyword, but with the
+    # need to support multiple keywords it was changed to
+    # access the keywords directly in CIAO 4.12.
+    #
     for filename in filenames:
-        v3("Looking for MJD_OBS keyword in {}".format(filename))
+        v3("Looking for MJD-OBS keyword in {}".format(filename))
+
+        hdr = get_keys_from_file(filename)
         try:
-            dmkeypar.punlearn()
-            dmkeypar(infile=filename, keyword='MJD_OBS', echo=False)
-            mjdobs = dmkeypar.rval
+            mjdobs = hdr['MJD-OBS']
+        except KeyError:
+            v3("Not found, looking for MJD_OBS")
+            try:
+                mjdobs = hdr['MJD_OBS']
+            except KeyError:
+                v3("Also not found, so calculating MJD_OBS value.")
 
-        except IOError:
-            v3("MJD_OBS keyword missing, so calculating MJD_OBS value.".format(filename))
+                # MJD(TT) = MJDREF(TT) + ( TIMEZERO + TIME ) / 86400
```

Comment:
As far as the change itself goes, I think it looks OK.  

But -- I just wonder if it would simplify things if we just ignored `MJD_|-OBS` and just use this fallback calculations?  Something to think about; not a lien on this PR.

FWIW: I am still worried we will run out of precision using doubles for `TIME` values and will need to change `TIMEZERO` at some point in which case just sorting on  `TSTART` won't be sufficient.

Reply:
I have a note for this routine that asks "can we not just sort on TSTART"? I think @nplee wrote this originally - can you remember why we sort on MJD-OBS here rather than some other time system?

Reply:
I'm going to merge this now, since we can think about changing the routine in another PR.